app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
It would be nice if we add some typing here for easier future refactoring and better IDE support, like this:
```suggestion
def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```